src/fields/multi_field.impl.hh

```diff
@@ -150,38 +142,72 @@ void MultiField<spacedim, Value>::set_up_components() {
     		sub_fields_[i_comp].shared_->input_name_ = name();
     	}
```

Comment:
Missing setup of the sub_field nema and input name if the comp_names_ is present.